.github/workflows/ci-workflow.yml

```
@@ -37,5 +37,5 @@ jobs:
       - name: 👏🏻 grant execute permission for gradlew
         run: chmod +x gradlew
 
-      - name: 🐘 build with Gradle
-        run: ./gradlew clean build -x test
\ No newline at end of file
+      - name: 🐘 build & test with Gradle
+        run: ./gradlew clean build
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)